dependencies/setupSundials.bat

```diff
@@ -0,0 +1,41 @@
+@ECHO OFF
+REM $Id$
```

Comment:
For new files, remove the old Subversion $Id$ thing, also the setup scripts should be added to the setupAll.bat and .sh